portfolio/src/main/webapp/script.js

```diff
@@ -50,8 +50,13 @@ function addPhoto() {
 }
 
 function getCommentSectionFromServer() {
-  fetch('/comment-section').then(response => response.json()).then((comments) => {
+  let commentsNumber = document.getElementById('comment-number').value;
+  let commentsOrder = document.getElementById('comment-order').value;
+  let url = `/comment-section?commentsnumber=${commentsNumber}&commentsorder=${commentsOrder}`;
```

Comment:
ultranit: Consider using punctuation in your url parameters: https://support.google.com/webmasters/answer/76329?hl=en

I don't have an opinion on whether this should be 'comments-number' 'comments_number' or 'commentsNumber'. The link above suggests dashes, but gives an example with underscores for query arguments, so it's a bit confusing.

Reply:
Done